modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -304,27 +305,54 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
 
             ///////////////////////////////////////////////////////////////////////
             //////check to see if candid/pscid and visit-label exist///////////////
+            //////        check to see if candidate is active        //////////////
+            //////        check to see if session is active          //////////////
             ///////////////////////////////////////////////////////////////////////
             if ((!empty($candid)) && (!empty($pscid)) && (!empty($visit_label))) {
-                $result = $db->pselectOne(
-                    "SELECT COUNT(*) FROM session s
-                     JOIN candidate c ON (c.candid=s.candid)
-                     WHERE s.Visit_label = :vlabel
-                     AND c.pscid = :pid AND c.candid = :cid",
+                $candidate = $db->pselectRow(
+                    "SELECT c2.candid AS candid_with_pscid, s.visit_label,
+                     c2.active AS candidate_active,
+                     s.active AS session_active
+                     FROM candidate c
+                     LEFT JOIN candidate c2
+                     ON (c.candid=c2.candid AND c2.pscid = :pid)
+                     LEFT JOIN session s
+                     ON (s.candid=c.candid and s.visit_label = :vlabel)
+                     WHERE c.candid = :cid",
                     [
                      ':cid'    => $candid,
-                     ':vlabel' => $visit_label,
                      ':pid'    => $pscid,
+                     ':vlabel' => $visit_label,
                     ]
                 );
 
-                if ($result == 0) {
-                    $errors[] = "Make sure you enter the CandID: $candid,".
-                                " PSCID: $pscid and the corresponding".
-                                " Visit-label: $visit_label. Candidate and".
-                                " corresponding visit label must already be" .
-                                " created in the database. Please refresh this".
-                                " page to return to the Imaging Uploader module";
+                if (sizeOf($candidate) == 0) {
+                    $errors[] = "The candidate associated with that CandID".
```

Comment:
in this case, it returns the following:
![screen shot 2018-05-09 at 11 21 32 am](https://user-images.githubusercontent.com/1402456/39823483-2635620c-537b-11e8-913f-e7a168633c45.png)

- I would tend to make the error logging consistent across the different cases maybe.
- Note sure the mri_file is needed in front of the error
- Would specify which CandID you are referring to in the error message (a bit like in the first row's error message but replacing DCCID by CandID)